spec/databag_spec.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'spec_helper'
+
+describe 'ucarp::data_bag' do
+  ALL_PLATFORMS.each do |p|
+    let(:chef_run) do
+=begin
+      ChefSpec::SoloRunner.new(p).converge(described_recipe)
+    end
+=end
```

Comment:
We have git commands to restore history; no reason to comment out code we don't intend to use.